core/ir/decodelib.c

```diff
@@ -128,7 +128,7 @@ get_thread_private_dcontext(void)
 void
 external_error(const char *file, int line, const char *msg)
 {
-    fprintf(stderr, "Usage error: %s (%s, line %d)\n", msg, file, line);
+    print_file(STDERR, "Usage error: %s (%s, line %d)\n", msg, file, line);
```

Comment:
Not sure drdecode links utils.c which has print_file.  The `STDERR` should work b/c it turns into either `our_stderr` or `dr_get_stderr_handle` and both are defined up above in this file.  Probably this should be `s/print_file/fprintf/`?

Reply:
Never mind, STDERR is the fileno and not the FILE*.  Plus, print_file is defined in this very file.